hypervisor/arch/x86/guest/vcpuid.c

```diff
@@ -40,10 +40,10 @@ static inline struct vcpuid_entry *local_find_vcpuid_entry(const struct acrn_vcp
 	return entry;
 }
 
-static inline struct vcpuid_entry *find_vcpuid_entry(const struct acrn_vcpu *vcpu,
+static inline const struct vcpuid_entry *find_vcpuid_entry(const struct acrn_vcpu *vcpu,
 					uint32_t leaf_arg, uint32_t subleaf)
```

Comment:
Then in caller side, will the following code needs type casting?
struct vcpuid_entry *entry =
                        find_vcpuid_entry(vcpu, leaf, subleaf);

Reply:
Yes. It already did.